lib/lightning_web/live/components/common.ex

```diff
@@ -25,12 +25,6 @@ defmodule LightningWeb.Components.Common do
            "Docker image tag found: '#{image}' tagged release build from #{commit}",
            :release}
 
-        # If running in docker and tag doesn't match :vsn, display image tag.
```

Comment:
Line 17 of this file already has behaviour to cover the case where `image == "edge"` so this clause is effectively identical to the clause below ` image != nil and image != vsn`.